pages/extensions/amp_example_preview/util/markdown_processor.py

```diff
@@ -1,73 +1,67 @@
-import os
-from grow.documents import document
-from .example_extractor import *
-from .preview import ExamplePreview
-from .constants import *
-
-import os
+from example_extractor import *
```

Comment:
I see it's easier to write, but can we instead use named imports here? 😉 That would make it more obvious where things are coming from.

Reply:
True :-)